hello-hwasan/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.4.1)
+add_library(native-lib SHARED native-lib.cpp)
+
+if(ANDROID_ABI STREQUAL "arm64-v8a")
```

Comment:
If we only support this on arm64, restrict the sample to arm64 only in the build.gradle using [ABI filters](https://developer.android.com/ndk/guides/abis#gc)